.github/workflows/gh-pages.yml

```diff
@@ -2,32 +2,50 @@ name: GitHub Pages
 
 on:
   push:
-    branches: [ master, audit, dev ]
+    branches: [ master, dev, feat/revamp-docs ]
```

Comment:
```suggestion
    branches: [ master, dev ]
```

Reply:
yup this will be removed just before merge